babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
isn't it affect the build somehow? it looks like a file for babel not only for testing...

Reply:
I think create react app uses preset babel config, this config is used to run the jest test

Reply:
Also our react sdk is using roll for build
![image](https://github.com/frontegg/frontegg-react/assets/106734943/2cb3245e-c847-4f6a-baf7-ee72f5064849)